### DISCUSSION TITLE
iOS杭州市民卡交通码URL_Scheme快捷打开 | Goalonez

### DISCUSSION THREAD

**Discussion by Goalonez** [Goalonez/goalonez.github.io#161] [Announcements]
https://blog.goalonez.site/blog/iOS%E6%9D%AD%E5%B7%9E%E5%B8%82%E6%B0%91%E5%8D%A1%E4%BA%A4%E9%80%9A%E7%A0%81URL_Scheme%E5%BF%AB%E6%8D%B7%E6%89%93%E5%BC%80.html 

 Goalonez Blog

**Comment by xiucaicoder** [comment]
我用的快捷启动 app，也是直接打卡市民卡 app，但不会打开交通码。切换成长辈版以后一下可以啦，感谢～

**Comment by ilonglong** [comment]
用的快捷指令，也是面临无法直接跳转进去的问题。感谢博主提供解决方案。

**Comment by luxuxl** [comment]
有点无语, 为什么不在钱包里开个交通卡, 公交车、地铁都能刷, 全国通用

**Comment by Goalonez** [comment]
> @luxuxl
> 有点无语, 为什么不在钱包里开个交通卡, 公交车、地铁都能刷, 全国通用

因为没有杭州通的优惠，另外至今 apple pay 和杭州通还没合作，只能走银联渠道（这个就比较看云闪付的活动了）。不过现在支付宝里有一码通，也就不需要市民卡这个 app 了。

**Comment by luxuxl** [comment]
> @Goalonez
> > @luxuxl
> > 有点无语, 为什么不在钱包里开个交通卡, 公交车、地铁都能刷, 全国通用
> 
> 因为没有杭州通的优惠，另外至今 apple pay 和杭州通还没合作，只能走银联渠道（这个就比较看云闪付的活动了）。不过现在支付宝里有一码通，也就不需要市民卡这个 app 了。

哈哈哈, 原来是有优惠吗, 不过我不在意这点优惠, 而且我估计目前是为了拉人, 等用户多了应该取消掉优惠了, 我还是比较怕麻烦的

**Comment by Kerrbty** [comment]
抓包拿下token，自己请求获取公交码最好
POST  https://traffic.iconntech.com/trafficPass/app/getQRCodeList.ext
```json
{
    "needHandle": "0",
    "accessToken": "xxxxxxxxxxx",
    "cardNo": "xxxxxxxx"
}
```
